### PR TITLE
ocamlPackages.ppxfind: dontStrip for all versions

### PR DESCRIPTION
**Pull request by anmonteiro**
###### Motivation for this change
I had to add `dontStrip` to support ppx_deriving (which uses ppxfind) when doing cross compilation to a static musl64 binary.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @vbgl

### CODE REVIEW

File: pkgs/development/ocaml-modules/ppxfind/default.nix [hunk 1]
```diff
@@ -12,14 +12,12 @@ buildDunePackage (rec {
 
 	buildInputs = [ ocaml-migrate-parsetree ];
 
+  dontStrip = true;
```
Comment (marsam) [inline pkgs/development/ocaml-modules/ppxfind/default.nix]:
```suggestion
  # Don't run the native `strip' when cross-compiling.
  dontStrip = stdenv.hostPlatform != stdenv.buildPlatform;
```

File: pkgs/development/ocaml-modules/ppxfind/default.nix [hunk 2]
```diff
@@ -12,14 +12,12 @@ buildDunePackage (rec {
...
+
 	meta = {
 		homepage = "https://github.com/diml/ppxfind";
 		description = "ocamlfind ppx tool";
 		license = lib.licenses.bsd3;
 		maintainers = [ lib.maintainers.vbgl ];
 	};
-} // (
```
Comment (marsam) [inline pkgs/development/ocaml-modules/ppxfind/default.nix]: is this change needed?

File: pkgs/development/ocaml-modules/ppxfind/default.nix (hunk 1 above)
Comment (anmonteiro) [reply pkgs/development/ocaml-modules/ppxfind/default.nix]: Oh this is great, thanks! Applied.

File: pkgs/development/ocaml-modules/ppxfind/default.nix (hunk 2 above)
Comment (anmonteiro) [reply pkgs/development/ocaml-modules/ppxfind/default.nix]: I suppose not in light of your other suggestion

**Comment by marsam** [comment]
Thanks!
I've added a missing argument, I hope you don't mind
@GrahamcOfBorg build flow

**Comment by anmonteiro** [comment]
@marsam not at all! thank you
